src/VirtualClient/VirtualClient.Main/profiles/PERF-REDIS-TLS.json

```diff
@@ -0,0 +1,226 @@
+{
+    "Description": "Redis Benchmark with TLS",
+    "Metadata": {
+        "RecommendedMinimumExecutionTime": "02:00:00",
+        "SupportedPlatforms": "linux-x64,linux-arm64",
+        "SupportedOperatingSystems": "Ubuntu",
+        "Notes_ConnectionLimits": "Each Memtier benchmark action will establish (--threads x --clients) connections against the Memcached server for each client instance/process. Ensure that the server-side OS does not place a limit on connections that might exceed this (e.g. ulimit -Sn on Linux)."
+    },
+    "Parameters": {
+      "ServerPort": 6397,
+      "ServerThreadCount": 4,
+      "ServerInstances": "{LogicalCoreCount}",
+      "ClientInstances": 1,
+      "Duration": "00:02:00"
+    },
+    "Actions": [
+        {
+            "Type": "RedisServerExecutor",
+          "Parameters": {
+            "Scenario": "redis_server_tls",
+            "PackageName": "redis",
+            "CommandLine": "--tls-cert-file {ScriptPath:redis}/RedisResources/redis.crt   --tls-key-file {ScriptPath:redis}/RedisResources/redis.key --tls-ca-cert-file {ScriptPath:redis}/RedisResources/ca.crt --protected-mode no --io-threads {ServerThreadCount} --maxmemory-policy noeviction --ignore-warnings ARM64-COW-BUG --save",
```

Comment:
Can we have all these in the code, if we already have the boolean option? It will confuse use as they might think having the boolean already works

Reply:
That's a valid point. That's one of the reasons why I haven't added IsTLSEnabled option as user changeable parameter. The confusion arises when user creates a custom profile.

Reply:
I would vote for just the boolean in parameter, and inject the arguments in the code